examples/third_party/openssl/openssl.BUILD

```diff
@@ -12,6 +12,12 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
+config_setting(
+    name = "windows",
+    values = {"cpu": "windows"},
```

Comment:
I think possibly this?
```suggestion
    values = {"cpu": "x64_windows"},
```

Reply:
I ended up doing something that felt cleaner but please check just in case.

Reply:
Works for me

Reply:
Rather than defining new config settings, couldn't you use @io_bazel_rules_rust//rust/platform:darwin, :windows, and/or conditions:default?

Reply:
That's what I started doing... until I undid the changes and ended up skipping CI for `@examples//complex_sys/...` on Windows as plenty of issues arise when building. Note that the `@examples//hello_sys/...` one is already skipped on Windows for similar reasons.

Reply:
(note: I did not convert the existing ones to preset platforms since I did not touch that part and want to avoid any additional fuss on this PR)